packages/enzyme-test-suite/test/adapter-utils-spec.jsx

```diff
@@ -137,6 +144,7 @@ describe('enzyme-adapter-utils', () => {
       <span />,
       <B />,
       <C />,
+      <RootFinder />,
```

Comment:
as mentioned previously; this shouldn't show up.

Reply:
Yup! This is part of the test data, not the expectation. I threw it in there to assert that it's getting filtered out like you've asked. 😊